tests/importable_test.py

```diff
@@ -94,6 +94,11 @@ def __str__(self) -> str:
         'vmware',
         'awx_plugins.inventory.plugins:vmware',
     ),
+    EntryPointParam(
+        'awx_plugins.inventory',
+        'vmware_esxi',
+        'awx_plugins.inventory.plugins:vmware_esxi',
+    ),
```

Comment:
Looks like this this checks for things in `[project.entry-points."awx_plugins.inventory"]` and not in `[project.entry-points."awx_plugins.inventory.supported"]`. You might need a new list of params for that. Seems like nobody added that in the past.